src/extension.ts

```diff
@@ -108,14 +110,13 @@ function startServer(context: ExtensionContext, javaHome: string) {
 
   context.subscriptions.push(
     commands.registerCommand("metals.restartServer", async () => {
-      const serverPid = client["_serverProcess"].pid;
-      await exec(`kill ${serverPid}`);
+      await client.sendRequest(ShutdownRequest.type);
+      client.sendNotification(ExitNotification.type);
       const showLogsAction = "Show server logs";
       const selectedAction = await window.showInformationMessage(
         "Metals Language Server killed, it should restart in a few seconds",
```

Comment:
While we're at this, I suggest we use something slightly less violent than "killed" 🗡 
How about "Terminated"?

Reply:
Good point, I kept "killed" when the server is truly `kill`ed. For the happy path the message is "Metals is restarting"